panther_lights_controller/src/controller_node.py

```diff
@@ -178,6 +185,10 @@ def _set_image_animation_cb(
 
         return 'success'
 
+    def _update_animations_cb(self, req: TriggerRequest) -> TriggerResponse:
+        self._update_animations()
+        return TriggerResponse(True, '')
```

Comment:
```suggestion
        return TriggerResponse(True, 'Animations updated successfully')
```

Also, shouldn't be there any try catch statement?